pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/style/AvoidNonExistentAnnotationsRule.java

```diff
@@ -0,0 +1,88 @@
+package net.sourceforge.pmd.lang.apex.rule.style;
```

Comment:
the rule seems like a better fit for the `errorprone` category (code that is likely to break) rather than a code style preference. Consider moving it there.